apps/extension/src/components/setting-sheet.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  HamburgerMenuIcon,
+  MixerHorizontalIcon,
+  PersonIcon,
+  SizeIcon,
+} from '@radix-ui/react-icons';
+import { Button, Sheet, SheetContent, SheetFooter, SheetHeader, SheetTrigger } from 'ui/components';
+import { useStore } from '../state';
+import { passwordSelector } from '../state/password';
+import { usePopupNav } from '../utils/navigate';
+import { PopupPath } from '../routes/popup/paths';
+import { PagePath } from '../routes/page/paths';
+
+const links = [
+  {
+    title: 'Settings',
+    icon: <MixerHorizontalIcon className='h-5 w-5 text-foreground' />,
+    href: PopupPath.SETTINGS,
+  },
+  {
+    title: 'Contact Information',
+    icon: <PersonIcon className='h-5 w-5 text-foreground' />,
+    href: PopupPath.SETTINGS_CONTACTS,
+  },
+];
+
+export const SettingSheet = () => {
+  const navigate = usePopupNav();
+  const { clearSessionPassword } = useStore(passwordSelector);
+  return (
+    <Sheet>
+      <SheetTrigger>
```

Comment:
It seems the sheet slides in nicely with an animation, but it does not slide out nicely. How do we get that?

Reply:
https://github.com/shadcn-ui/ui/issues/72. This issue is still open, but the solution is to add "data-[state=closed]:fill-mode-forwards data-[state=open]:fill-mode-forwards"  where animation use. maybe you will come across such a bug someday